debian/control

```diff
@@ -38,6 +38,21 @@ Depends: ${misc:Depends}, ${shlibs:Depends}, libqubesdb
 Description: This is securedrop Qubes proxy service
  This package provides the network proxy on Qubes to talk to the SecureDrop server.
 
+Package: securedrop-qubesdb
+Section: admin
+Architecture: all
+Depends: ${misc:Depends}, python3, python3-qubesdb
+Description: Tools for configuring non-Qubes-aware applications from QubesDB.
+ This package provides tools for configuring non-Qubes-aware applications from
+ QubesDB.
+
+Package: securedrop-whonix-config
+Section: admin
+Architecture: all
+Depends: ${misc:Depends}, securedrop-qubesdb, tor
```

Comment:
There's still going to be a bootstrapping problem on this pull request, since `securedrop-qubesdb` isn't published yet for `piuparts` to pull.

Reply:
Ugh, yeah, that doesn't work. I think we should just copy all the packages into the piuparts container, and set up an apt source of `file:///packages` with the help of [`dpkg-scanpackages(1)`](https://manpages.debian.org/bookworm/dpkg-dev/dpkg-scanpackages.1.en.html). Let me try...

Reply:
Try cherry-picking https://github.com/freedomofpress/securedrop-client/commit/d9eca8825da9a250d9b381059c425e56a22224eb?

Reply:
Very elegant!  `piuparts` complains both here and directly on d9eca88 about:

```
ERROR: FAIL: After purging files have been modified:
  /var/lib/dpkg/info/fontconfig-config.list	 not owned
  /var/lib/dpkg/info/fonts-dejavu-core.list	 not owned
  /var/lib/dpkg/info/libtirpc-common.list	 not owned
  /var/lib/dpkg/info/readline-common.list	 not owned
```

Reply:
Wound up cherry-picking a68641b48ceb51f468af7b0ae23fcce9abbe5cc8 per your out-of-band recommendation and then fixing a path problem in 8aab71c4566ae231fdde9145fea96132b8777cac.  Feel free to squash it down (or I can) if you're happy with it.